timely_beliefs/beliefs/classes.py

```diff
@@ -84,7 +84,8 @@ def __init__(
         elif "belief_time" in kwargs:
             belief_time = tb_utils.enforce_tz(kwargs["belief_time"], "belief_time")
             self.belief_horizon = (
-                self.sensor.knowledge_time(self.event_start) - belief_time
+                self.sensor.knowledge_time(self.event_start, self.event_resolution)
```

Comment:
I see that the event resolution of a belief trumps the event resolution of a sensor. In other words, the sensor's event resolution is used as fallback, right?

Is that documented? In the Readme, I can't even find information about event resolution of beliefs...

Reply:
The documentation does contain information about even resolutions, but does not yet explain any distinction between the event resolution of the BeliefsDataFrame and the event resolution of the sensor. I find your written questions very useful for writing such new documentation. I'll add it.

Also, you helped me uncover a bug, because `sensor.knowledge_time()` was still sometimes called without specifying the updated event resolution.